luajit.krun

```diff
@@ -0,0 +1,96 @@
+import os
+from krun.vm_defs import (LuaVMDef)
+from krun import EntryPoint
+from krun.util import fatal
+
+# Who to mail
+MAIL_TO = []
+
+# Maximum number of error emails to send per-run
+#MAX_MAILS = 2
+
+DIR = os.getcwd()
+
+HEAP_LIMIT = 2097152  # KiB
+STACK_LIMIT = 8192  # KiB
+
+# Variant name -> EntryPoint
+VARIANTS = {
+    "default-lua": EntryPoint("bench.lua"),
+}
+
+ITERATIONS_ALL_VMS = 5  # Small number for testing.
+
+VMS = {
+    'Normal': {
+        'vm_def': LuaVMDef("builds/luajit"),
+        'variants': ['default-lua'],
+        'n_iterations': ITERATIONS_ALL_VMS,
+    },
+    'GC64': {
+        'vm_def': LuaVMDef("builds/luajit_gc64"),
+        'variants': ['default-lua'],
+        'n_iterations': ITERATIONS_ALL_VMS,
+    },
+    'NoJIT': {
+        'vm_def': LuaVMDef("builds/luajit_nojit"),
+        'variants': ['default-lua'],
+        'n_iterations': ITERATIONS_ALL_VMS,
+    },
+    'DualNum': {
+        'vm_def': LuaVMDef("builds/luajit_dualnum"),
+        'variants': ['default-lua'],
+        'n_iterations': ITERATIONS_ALL_VMS,
+    },
+}
+
+BENCHMARKS = {
+    "binarytrees" : 1,
+    "nbody" : 1,
+    "fasta": 1,
+    "richards": 1,
+    "spectralnorm": 1,
+    "fannkuch_redux": 1,
+    "md5": 1,
+    "series": 1,
+    "luacheck_parser": 1,
+    "luacheck": 1,
+    "capnproto_encode": 1,
+    "capnproto_decode": 1,
+    "jsonlua_encode": 1,
+    "jsonlua_decode": 1,
+}
+
+# list of "bench:vm:variant"
+SKIP=[
+    #"*:C:*",
+    #"*:PyPy:*",
+]
+
+N_EXECUTIONS = 1  # Number of fresh processes.
+
+# No. of seconds to wait before taking the initial temperature reading.
+# You should set this high enough for the system to cool down a bit.
+# The default (if omitted) is 60 seconds.
+TEMP_READ_PAUSE = 2
+
+# Commands to run before and after each process execution
+#
+# Environment available for these commands:
+#   KRUN_RESULTS_FILE: path to results file.
+#   KRUN_LOG_FILE: path to log file.
+#   KRUN_ETA_DATUM: time the ETA was computed
+#   KRUN_ETA_VALUE: estimated time of completion
+#PRE_EXECUTION_CMDS = ["sudo service cron stop"]
+#POST_EXECUTION_CMDS = ["sudo service cron start"]
+
+# CPU pinning (off by default)
+ENABLE_PINNING = True
+
+# Lower and upper bound for acceptable APERF/MPERF ratios
+AMPERF_RATIO_BOUNDS = 0.995, 1.005
+
+# Rough rate of change in APERF per second above which a core is considered busy.
+# For many machines this is simply the base clock frequency, but generally
+# speaking, is undefined, so you need to check on a per-machine basis.
+AMPERF_BUSY_THRESHOLD = 3.4 * 1000 * 1000 * 1000 / 1000  # 3.4 GHz / 1000
```

Comment:
I'd skip this section too until we get a proper benchmarking machine up and running.

Reply:
You can leave XXX markers as reminders if you like...